### DISCUSSION TITLE
Documentation gives all the detail but no high level summary

### DISCUSSION THREAD

**Discussion by BryanCrotaz** [wixtoolset/issues#7777] [Questions]
I've just read through all the docs at https://wixtoolset.org/docs and I'm none the wiser how to create an installer for my solution.

I gather I have to run Heat, but according to the docs I shouldn't do that because it doesn't give the correct results, However it doesn't say what the preferred way of doing it is. Apparently I have to "burn" some variables in, but don't know how to tell burn which projects to install.

Where is the step 1-5 "My first Wix installer" docs page? I can't find it.

**Comment by robmen** [comment]
At this time, the WiX Toolset site is mostly reference material. FireGiant maintains a [more introductory step-by-step tutorial](https://www.firegiant.com/docs/wix/tutorial/) and topics continue to be added over time.

Perhaps there should be a link from wixtoolset.org to the FireGiant tutorial somewhere?

**Reply by BryanCrotaz** [reply]
I've been asked to contribute to docs with my learnings and feedback as a virgin user (I used Wix v1 or possibly 2 a loooooong time ago). But I can't find which repo they are in. The above tutorial link for example, how do I contribute? Or if I want to add information to https://wixtoolset.org/docs/tools/heat/?
